spring-cloud-gcp-samples/spring-cloud-gcp-data-multi-sample/src/test/java/com/example/MultipleDataModuleTest.java

```diff
@@ -44,6 +47,14 @@
 	@Autowired
 	PersonRepository datastorePersonRepository;
 
+	@BeforeClass
+	public static void checkToRun() {
+		assumeThat(
+				"Spanner integration tests are disabled. "
+						+ "Please use '-Dit.spanner=true' to enable them. ",
+				System.getProperty("it.spanner"), is("true"));
```

Comment:
Do you need a second assumption that `it.datastore` is enabled as well?

Reply:
yes good point